embassy-mspm0/src/lib.rs

```diff
@@ -18,6 +18,7 @@ pub mod gpio;
 pub mod i2c;
 pub mod timer;
 pub mod uart;
+pub mod watchdog;
```

Comment:
We should match the datasheet/reference manual and use `wwdt` for the module name. All the other embassy-hals to do this. Even nRF with its unusual `twim` module name. 

The type names inside the module can stay as is.